x/wasm/internal/keeper/keeper.go

```diff
@@ -138,15 +138,11 @@ func (k Keeper) Instantiate(ctx sdk.Context, codeID uint64, creator sdk.AccAddre
 
 	// instantiate wasm contract
 	gas := gasForContract(ctx)
-	res, err := k.wasmer.Instantiate(codeInfo.CodeHash, params, initMsg, prefixStore, cosmwasmAPI, querier, gas)
+	res, gasUsed, err := k.wasmer.Instantiate(codeInfo.CodeHash, params, initMsg, prefixStore, cosmwasmAPI, querier, gas)
+	consumeGas(ctx, gasUsed)
```

Comment:
Shouldn't this be divided by the `GasMultiplier`?

Reply:
That is done inside `consumeGas`

Reply:
Ah right, I forgot that when reviewing the code